tasks/src/robocorp/tasks/__init__.py

```diff
@@ -172,4 +194,5 @@ def get_current_task() -> Optional[ITask]:
     "get_current_task",
     "ITask",
     "Status",
+    "inject_truststore",
```

Comment:
Just because we plan to re-use this utility function in **browser**'s CLI as well, I wouldn't expose it to the library's public interface.

Instead I would move the `inject_truststore()` definition right under _cli.py_ (as it was before), then in **browser** will do the `from robocorp.tasks.cli import inject_truststore` so I can trigger the injection there as well symmetrically to how it was done in the **tasks**.

My main worry is this: The **tasks** library user won't ever need to import and use such `inject_truststore` during the library usage under the automation development, so we shouldn't expose that to the public interface.

Do you see any limitation with my proposed solution or clear benefit given your current one?

Reply:
This is a good point. I think we can go with your proposal. I don't see any limitations or extra benefits from my approach.